docs/BMI-Installation.md

```diff
@@ -0,0 +1,279 @@
+# BMI
+
+The Bare Metal Imaging (BMI) is a core component of the Massachusetts Open Cloud and a image management system(ims) that  
+* provisions numerous nodes as quickly as possible while preserving support for multitenancy using Hardware as a Service (HaaS) and  
+* introduces the image management techniques that are supported by virtual machines, with little to no impact on application performance.  
+
+***
+## Installation
+
+This document describes how to setup BMI
+
```

Comment:
I advise adding @sirushtim's script as the first way to install for developers. Then the manual installation can continue